site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -208,6 +208,16 @@ private function editPage(Gradeable $gradeable, $semester, $course, $nav_tab = 0
             }
         }
         // $this->inherit_teams_list = $this->core->getQueries()->getAllElectronicGradeablesWithBaseTeams();
+        $template_list = $this->core->getQueries()->getAllGradeablesIdsAndTitles();
+
+        $max_points = 0;
+        if ($gradeable->getDependsOn() !== null) {
+            $depend_gradeable = $this->tryGetGradeable($gradeable->getDependsOn());
```

Comment:
It looks like this function is only being called from a HTML-returning route, so false should be passed in as the second parameter to disable JSON rendering.